TrackController/src/trackCtrl/control/Main.java

```diff
@@ -13,10 +13,12 @@ public void start(Stage primaryStage) throws Exception {
     Parent root = FXMLLoader.load(getClass().getResource("../view/trackCtrl.fxml"));
```

Comment:
This line breaks when I try to run because Main.java is in Control package and this path is not correct.